libvips/foreign/exif.c

```diff
@@ -78,6 +78,27 @@
 
 #include "pforeign.h"
 
+// render an entry as a utf8 string
+static char *
+entry_to_s(ExifEntry *entry)
+{
+	// some extra space for conversion to string ... this can be quite a bit
+	// for formats like float
+	size_t max_size = entry->size * 5;
```

Comment:
Can we trust `entry->size`? Perhaps worth adding a limit of our own.

Reply:
Good idea!